src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -157,6 +162,77 @@ class SVTResourceResolver: AzSKRoot {
+        # fetching the repository count of a project
+        try{
+            $resourceURL = "https://dev.azure.com/$($this.organizationName)/$($projectName)/_apis/git/repositories?api-version=6.0"
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL) ;
+            # $this.AddSVTResource("Repos", $projectName, "ADO.Repo", "organization/$organizationId/project/$projectId", $null, "");
+            $projectData['repositories'] = $responseList.Length
+            # fetching the testplan count of a project
+            $resourceURL = "https://dev.azure.com/$($this.organizationName)/$($projectName)/_apis/testplan/plans?api-version=6.0-preview.1"
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL) ;
+            $projectData['testplan'] = $responseList.Length
+            # fetching the taskgroups count of a project
+            $resourceURL = "https://dev.azure.com/$($this.organizationName)/$($projectName)/_apis/distributedtask/taskgroups?api-version=6.0-preview.1"
+            $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL) ;
+            $projectData['taskgroups'] = $responseList.Length
+            # fetch the builds count if provided builds not equals to *
+            if($projectData['build'] -eq -1) {
+                $resourceURL = ""
+                if ([string]::IsNullOrEmpty($topNQueryString)) {
+                    $topNQueryString = '&$top=10000'
+                    $resourceURL = ("https://dev.azure.com/{0}/{1}/_apis/build/definitions?api-version=4.1&queryOrder=lastModifiedDescending" +$topNQueryString) -f $($this.SubscriptionContext.SubscriptionName), $projectName;
+                }
+                else {
+                    $resourceURL = ("https://dev.azure.com/{0}/{1}/_apis/build/definitions?api-version=4.1" +$topNQueryString) -f $($this.SubscriptionContext.SubscriptionName), $projectName;                               
+                }
+                $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL);
+                $projectData['build'] = $responseList.Length;
+            }
+            # fetch the release count if provided builds not equals to *
+            if($projectData['release'] -eq -1) {
+                $resourceURL = ("https://vsrm.dev.azure.com/{0}/{1}/_apis/release/definitions?api-version=4.1-preview.3" +$topNQueryString) -f $($this.SubscriptionContext.SubscriptionName), $projectName;
+                $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL);
+                $projectData['release'] = $responseList.Length;
+            }
+            # fetch the agent pools count
+            if($projectData["agentpools"] -eq -1) {
+                $agentPoolsDefnURL = ("https://dev.azure.com/{0}/{1}/_settings/agentqueues?__rt=fps&__ver=2") -f $($this.SubscriptionContext.SubscriptionName), $projectName;
```

Comment:
Try to use existing code(even if small modification required), avoid writing same code in multiple places, we already have api calls for resources.